aya/src/programs/tc.rs

```diff
@@ -180,14 +175,16 @@ pub fn qdisc_add_clsact(if_name: &str) -> Result<(), io::Error> {
 pub fn qdisc_detach_program(
     if_name: &str,
     attach_type: TcAttachType,
-    name: &str,
+    name: &CStr,
```

Comment:
Please revert this. This is a public helper and requiring callers to pass `&CStr` instead of `&str` makes for a worse API.

Reply:
I've added a _fast variant for those who wish to opt into the less ergonomic API to avoid the allocation.

Reply:
I don't think it's worth having (and having to support) extra API for this to avoid a memcpy of a handful of bytes

Reply:
Alright, I'll make this private then.